pkgs/servers/http/apache-modules/mod_ca/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
+ suffix = ".tar.gz";
+
+ meta = with stdenv.lib; {
+   description = "RedWax CA service modules.";
+   version = "0.2.1";
+
+   homepage = "https://redwax.eu";
+   license = licenses.asl20;
+   platforms = platforms.unix;
+   maintainers = with maintainers; [ dirkx ];
+ };
+
+ src = fetchurl {
+   url = "${baseurl}${name}-${meta.version}${suffix}";
+   sha256 = "1pxapjrzdsk2s25vhgvf56fkakdqcbn9hjncwmqh0asl1pa25iic";
+ };
+
+ buildInputs = [ gnused coreutils pkgconfig apacheHttpd openssl openldap ];
+
+ configurePlatforms = [];
+ configureFlags = [
+       "--with-apxs=${apacheHttpd.dev}/bin/apxs"
```

This seems to be a good place to mention that indentation in packages should be a consistent 2 spaces per level of indentation.

---

So we do something a bit unusual and that is we have a main apache module (mod_ca) and then modules that need both the apache dl/apxs -and- mod-ca. So that means that you are not just runtime pulling dl's from any path - but also includes during build. So this is a bit like why you somethings see a libperl or libpython next to a mod_perl stub.

Regardless - we cannot install them in the APXS provided location for modules -- as this one is private to httpd (/nix/store/m15l5xmr2s02sji9yh91nip6b6hrr8r4-apache-httpd-2.4.41/modules/mod_include.so).

But no problem to change this to modules - and then do the same 'SED'

---

fixed all indentations too.

---

```suggestion
    "--with-apxs=${apacheHttpd.dev}/bin/apxs"
```
@dirkx this line should only have 4 spaces as it is only 2 levels of indentation in.